spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu'
+
+describe Menu do
+  it "creates a new instance of menu" do
+    expect(subject).to be_an_instance_of Menu
+  end
+
+  it "initializes with a full menu" do
+    expect(subject.dishes.length).to eq 5
```

Comment:
What if `subject.dishes` was [1,2,3,4,5], would this test pass? Should it? There's probably a stronger assertion we can make here

Reply:
Test changed to ensure that subject.dishes includes ("pizza", "burger", "steak", "ice cream", "chow mein")